### PR TITLE
Add Random mod to legacy mod lists

### PR DESCRIPTION
**Pull request by cl8n**
not important at all but just noticed this was missing, when working on the other mods thing.

### CODE REVIEW

File: app/Libraries/Mods.php [hunk 1]
```diff
@@ -34,6 +34,7 @@ class Mods
         1 << 13 => 'AP' ,
         1 << 14 => 'PF' ,
         1 << 20 => 'FI' ,
+        1 << 21 => 'RD' ,
```
Comment (nanaya) [inline app/Libraries/Mods.php:37]: where did you get this 21 from?

Comment (cl8n) [reply app/Libraries/Mods.php:37]: <https://github.com/ppy/osu-api/wiki#mods>, and just to be sure I found this very old lazer version that has Mods enum 
<https://github.com/ppy/osu/blob/45f500920cd2338aef0f0b3158753d026e36100f/osu.Game/Modes/Mod.cs#L235-L236>

Comment (peppy) [reply app/Libraries/Mods.php:37]: I guess the real question is whether this is displayed anywhere, and if not, should it be included? Rationale is that there's other assigned mods in stable's code which will never be displayed by osu-web. And at this point in migrating to a newer system that doesn't use these bitwise values, I'd argue whether this is a good direction or not.

```csharp
    [Flags]
    public enum Mods
    {
        None                   = 0,
        NoFail                 = 1 << 0,
        Easy                   = 1 << 1,
        TouchDevice            = 1 << 2,
        Hidden                 = 1 << 3,
        HardRock               = 1 << 4,
        SuddenDeath            = 1 << 5,
        DoubleTime             = 1 << 6,
        Relax                  = 1 << 7,
        HalfTime               = 1 << 8,
        Nightcore              = 1 << 9,
        Flashlight             = 1 << 10,
        Autoplay               = 1 << 11,
        SpunOut                = 1 << 12,
        Relax2                 = 1 << 13,
        Perfect                = 1 << 14,
        Key4                   = 1 << 15,
        Key5                   = 1 << 16,
        Key6                   = 1 << 17,
        Key7                   = 1 << 18,
        Key8                   = 1 << 19,
        FadeIn                 = 1 << 20,
        Random                 = 1 << 21,
        Cinema                 = 1 << 22,
        Target                 = 1 << 23,
        Key9                   = 1 << 24,
        KeyCoop                = 1 << 25,
        Key1                   = 1 << 26,
        Key3                   = 1 << 27,
        Key2                   = 1 << 28,
        ScoreV2                = 1 << 29,
        Mirror                 = 1 << 30,

        KeyMod = Key1 | Key2 | Key3 | Key4 | Key5 | Key6 | Key7 | Key8 | Key9 | KeyCoop,
        FreeModAllowed = NoFail | Easy | Hidden | HardRock | SuddenDeath | Flashlight | FadeIn | Relax | Relax2 | SpunOut | KeyMod | Mirror,
        ScoreIncreaseMods = Hidden | HardRock | DoubleTime | Flashlight | FadeIn
    }
```

**Review by nanaya — APPROVED** [review]
The icon is probably useful to have at least.

**Pull request merged**